packages/studio-base/src/panels/Map/config.ts

```diff
@@ -9,12 +9,13 @@ import { SettingsTreeFields, SettingsTreeNodes } from "@foxglove/studio";
 
 // Persisted panel state
 export type Config = {
+  center: undefined | { lat: number; lon: number };
```

Comment:
nit -- or is there a reason you don't do this?
```suggestion
  center?: { lat: number; lon: number };
```